.github/workflows/l4lb.yaml

```diff
@@ -0,0 +1,63 @@
+name: Cilium L4LB XDP
+
+# Any change in triggers needs to be reflected in the concurrency group.
+on:
+  issue_comment:
+    types:
+      - created
+  # Run every 6 hours
+  schedule:
+    - cron:  '0 2/6 * * *'
+  ### FOR TESTING PURPOSES
+  # pull_request:
+  #  types:
+  #    - "labeled"
+  ###
+
+concurrency:
+  group: "${{ github.workflow }} ${{ github.event.issue.pull_request.url || 'scheduled' }} ${{ github.event.comment.body }}"
```

Comment:
That last part (`github.event.comment.body`) will generate errors when triggered by large comments. See https://github.com/cilium/cilium/commit/6440f293d27719d76d239ba12e3dd91914cf5548.

Reply:
Thanks for writing details git commit messages. It helped to understand the reasoning and the issues behind the concurrency groups.